Retrovizor.Domain/Repositories/Implementations/StudentExamRepository.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Retrovizor.Data.Entities;
+using Retrovizor.Data.Entities.Models;
+using Retrovizor.Domain.Repositories.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Retrovizor.Domain.Repositories.Implementations
+{
+    public class StudentExamRepository : IStudentExamRepository
+    {
+        StudentExamRepository(RetrovizorContext context)
+        {
+            _context = context;
+        }
+        private readonly RetrovizorContext _context;
+
+        public List<StudentExam> GetAllStudentExams()
+        {
+            return _context.StudentExams.ToList();
+        }
+        public bool AddStudentExam(StudentExam studentExamToAdd)
```

Comment:
isto ka i gori sta se tice 2 eventa u isto vrime. Uz to triba pripazit i da ne mozes imat event i exam u isto vrime tako da triba napravit filter i za exam i za event na oba mista

Reply:
Exam nema vrime, predstavlja fizicki ispit u smislu bodova(i eventualno pitanja), a sam ispit u rasporedu je zapravo event. Provjera nije potrebna.